src/configurations/nf/organizations.ts

```diff
@@ -23,49 +34,68 @@ export const organizationDetailsPageConfig: DetailsPageProps = {
       title: 'Funded Studies',
       columnName: 'fundingAgency',
       tableSqlKeys: ['fundingAgency'],
+      tabIndex: 0,
     },
     {
       name: 'CardContainerLogic',
       props: {
-          sql: datasetsSql,
-        type: SynapseConstants.DATASET,
+        sql: publicationsSql,
+        ...publicationsCardConfiguration,
+        sqlOperator: 'LIKE',
       },
+      title: 'Publications',
       columnName: 'fundingAgency',
-      title: 'Datasets',
       tableSqlKeys: ['fundingAgency'],
+      tabIndex: 0,
     },
     {
-      name: 'StandaloneQueryWrapper',
+      name: 'Markdown',
       props: {
-        facetAliases,
-        title: 'Files',
-        rgbIndex: 1,
-        facet: 'assay',
-        sql: filesSql,
+        ownerId: 'syn22272075',
+        wikiId: '604853',
       },
-      title: 'Files',
-      columnName: 'fundingAgency',
-      tableSqlKeys: ['fundingAgency'],
+      title: 'Funding Impact',
+      standalone: true,
+      tabIndex: 0,
     },
     {
       name: 'CardContainerLogic',
       props: {
-          sql: publicationsSql,
-        ...publicationsCardConfiguration,
-        sqlOperator: 'LIKE',
+          sql: datasetsSql,
+        type: SynapseConstants.DATASET,
       },
-      title: 'Publications',
       columnName: 'fundingAgency',
+      title: 'Datasets',
       tableSqlKeys: ['fundingAgency'],
+      tabIndex: 1,
     },
     {
-      name: 'Markdown',
+      name: 'QueryWrapperPlotNav',
+      tabIndex: 1,
       props: {
-        ownerId: 'syn22272075',
-        wikiId: '604853',
+        rgbIndex: 8,
+        shouldDeepLink: false,
+        sql: filesSql,
+        visibleColumnCount: 7,
+        sqlOperator: 'LIKE',
+        tableConfiguration: {
+          showAccessColumn: true,
+        },
+        name: 'Data Files',
+        facetAliases,
+        searchConfiguration: {
+          searchable: [
+            'datasetName',
+            'summary',
+            'studyName',
+            'diseaseFocus',
+            'manifestation',
+            'fundingAgency',
+          ],
+        },
```

Comment:
The `searchConfiguration` is copied from the study details page settings. Do we need this `searchConfiguration` in organization/funder details page?

Reply:
Likely yes (to define the search columns that make sense).  Ideally this would be defined once, and reused in all of these different places (so in the future when it needs to be modified, we don't have to hunt them all down!)

Reply:
Good idea! I moved the search config settings to `commonProps.ts``